flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/core/testutils/FlinkAssertions.java

```diff
@@ -117,7 +117,7 @@ private FlinkAssertions() {}
      *                  .hasMessageContaining(containsMessage));
      * }</pre>
      */
-    public static ThrowingConsumer<? extends Throwable> anyCauseMatches(String containsMessage) {
+    public static ThrowingConsumer<? super Throwable> anyCauseMatches(String containsMessage) {
```

Comment:
put this into a separate commit, because it is a core change

Reply:
+1 just as FYI no one is using this method (also because without this change is unusable :smile: )